RadixWallet/SargonExtensions/SargonSecureStorageDriver.swift

```diff
@@ -61,6 +61,10 @@ final class SargonSecureStorage: SecureStorageDriver {
 		}
 	}
 
+	func containsDataForKey(key: SecureStorageKey) async throws -> Bool {
+		false
```

Comment:
missed implementation?

Reply:
not used by host yet I think. Sargon contains newer APIs than hosts needs. Mathias is working on this part of FactorSource work. So should be safe to return `false` here since it is in fact never called from Sargon.